src/utils/questionnaireBuilderUtils.ts

```diff
@@ -204,74 +192,80 @@ export enum ValidationCondition {
   TITLE_EMPTY = "titleEmpty",
   DESCRIPTION_EMPTY = "descriptionEmpty",
   NO_QUESTIONS = "noQuestions",
-  NO_CORRECT_ANSWERS = "noCorrectAnswers", 
+  QUESTIONS_AND_ANSWERS_EMPTY = "questionsAndAnswersEmpty",
+  QUESTIONS_EMPTY = "emptyQuestions",
```

Comment:
It seems to add an unnecessary layer to me that we have this enum to handle validation conditions and then later convert it to a string. Why not just set it as a string here straight away?
It might be that it causes issues to use the localization library hook in a .ts file, if so then this is fine as it is.

Reply:
well we can access the strings in getQuestionValidationTooltipMessage in this file so don't see why we can't here. 
I could be mistaken but it seems to me setting the string rather than an enum here simplifies things